manifests/i/iLovePDF/iLovePDFDesktop/1.4.1.0/iLovePDF.iLovePDFDesktop.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: iLovePDF.iLovePDFDesktop
+PackageVersion: 1.4.1.0
+PackageLocale: en-US
+Publisher: iLovePDF
```

```suggestion
Publisher: iLovePDF.S.L
```
This must match Control Panel.